composer.json

```
@@ -114,6 +119,7 @@
         "roave/security-advisories": "dev-latest",
         "symfony/console": "^5.2.3",
         "tecnickcom/tcpdf": "^6.4.4",
+        "thecodingmachine/safe": "1.3.3.1",
```

```suggestion
        "thecodingmachine/safe": "^1.3.3.1",
```
That would be better if you release further fixes